tests/unit/test-class-wc-payments-account.php

```diff
@@ -843,8 +843,6 @@ public function test_maybe_handle_onboarding_init_embedded_kyc() {
 			->expects( $this->never() )
 			->method( 'redirect_to_onboarding_wizard' );
 
-		update_option( '_wcpay_feature_embedded_kyc', '1' );
```

Comment:
nitpick: you should also delete this from `tests/unit/test-class-wc-payments-account.php` - `tear_down` function

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/9445/commits/7ca58a8f8f51d5df09e4a187d5cf270f73396667